src/GenAPI/Microsoft.DotNet.GenAPI/SyntaxRewriter/OneLineStatementSyntaxRewriter.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+
+namespace Microsoft.DotNet.GenAPI.SyntaxRewriter
+{
+    /// <summary>
+    /// Removes multiline formatting for one line statements:
+    /// ```int Execute() {
+    ///     throw null;
+    /// }```
+    /// =>
+    /// ```int Execute() { throw null; }```
+    /// </summary>
+    public class OneLineStatementSyntaxRewriter : CSharpSyntaxRewriter
```

Comment:
As this type inherits from `CSharpSyntaxRewriter`, should its name express that, i.e. by calling it `SingleLineStatementCSharpSyntaxRewriter`?

Reply:
updated